doc/user/content/sql/create-connection.md

```diff
@@ -149,34 +234,98 @@ CREATE CONNECTION pg_connection TO POSTGRES (
 );
 ```
 
-### SSH tunneling {#postgres-ssh}
```

Comment:
Not sure about adding it in this way, but I can reorg it later. We'll also need a PrivateLink example for Postgres, right?